react-native/react/native/notification-listeners.desktop.js

```diff
@@ -7,74 +6,76 @@ import type {FSNotification} from '../../react/constants/types/flow-types'
 // notification listeners, store the sentNotifications map in it.
 var sentNotifications = {}
 
-export default {
-  'keybase.1.NotifySession.loggedOut': () => {
-    notify('Logged Out')
-  },
-  'keybase.1.NotifyFS.FSActivity': params => {
-    const notification: FSNotification = params.notification
+export default function (notify) {
```

Comment:
The only change in this file is that it is now a function that takes in a `NotifyFn` and returns a map of listeners (so just these two lines really). The rest is whitespace changes.